ext/hash/hash.stub.php

```diff
@@ -2,13 +2,13 @@
 
 /** @generate-function-entries */
 
-function hash(string $algo, string $data, bool $raw_output = false): string|false {}
+function hash(string $algo, string $data, bool $binary = false): string|false {}
 
-function hash_file(string $algo, string $filename, bool $raw_output = false): string|false {}
+function hash_file(string $algo, string $filename, bool $binary = false): string|false {}
 
-function hash_hmac(string $algo, string $data, string $key, bool $raw_output = false): string|false {}
+function hash_hmac(string $algo, string $data, string $key, bool $binary = false): string|false {}
 
-function hash_hmac_file(string $algo, string $data, string $key, bool $raw_output = false): string|false {}
+function hash_hmac_file(string $algo, string $data, string $key, bool $binary = false): string|false {}
 
 function hash_init(string $algo, int $options = 0, string $key = ""): HashContext {}
```

Comment:
The `$options` parameter appears to be intended as a flags bitmask, even though it currently features just one flag. These parameters are more commonly named `$flags` elsewhere.